__tests__/app.test.js

```diff
@@ -21,12 +20,52 @@ describe('/api/categories', () => {
             .expect(200)
             .then(({body}) => {
                 expect(Array.isArray(body.categories)).toBe(true)
-                expect(body.categories.length > 0)
+                expect(body.categories.length = 4)
                 body.categories.forEach((category) => {
                     expect(category).toHaveProperty('slug', expect.any(String))
                     expect(category).toHaveProperty('description', expect.any(String))
                 })
             })
         })
     })
+})
+
+describe('/api/reviews', () => {
+    describe('GET by review_id', () => {
+        test('200: responds with an object containing a key of reviews and a value of an array containing the review with the provided review_id', () =>{
+            return request(app)
+            .get('/api/reviews/1')
+            .expect(200)
+            .then(({body}) => {
+                expect(body.reviews).toEqual([{
```

Comment:
We would want this review back as an object on a key of 'review', so the body would look something like:
`{
  review: {
    review_id: 1,
    ...
  }
}`

Reply:
Right now you have it on a key of 'reviews' and its an array of length 1 - just a bit of data manipulation in your api needed.

Reply:
done